### ISSUE TITLE
excluded paths: server functions not found

### ISSUE THREAD

**Issue by yuhime** [leptos-rs/leptos#3175]
**Server function not found after insert it on excluded paths**
now the server functions are successfully excluded but also removed from `REGISTERED_SERVER_FUNCTIONS` this leads to the fact that handle_server_fns no longer works over the excluded functions, is it possible to exclude them only from automatic path generation?

**To Reproduce**
```rust
    let routes = generate_route_list_with_exclusions(
        {
            let options = leptos_options.clone();
            move || shell(options.clone())
        },
        Some(vec!["/api/my_server_functions".into()]),
    );
    
    Router::new().leptos_routes(&leptos_options, routes, {
            let options = leptos_options.clone();
            move || shell(options.clone())
        }) .route(
            "/api/my_server_functions",
            post(handle_server_fn)
        )
```
```rust
Could not find a server function at the route /api/my_server_functions.

It's likely that either
 1. The API prefix you specify in the `#[server]` macro doesn't match the prefix at which your server function handler is mounted, or
2. You are on a platform that doesn't support automatic server function registration and you need to call ServerFn::register_explicit() on the server function type, somewhere in your `main` function.
```

**Expected behavior**
handle_server_fns should be able to find the function so that it can be handled with axum together with the route and the layer

**Comment by gbj** [comment]
Duplicate of #3150.

**Comment by gbj** [comment]
Fixed by #3176
